folly/detail/IPAddress.h

```diff
@@ -283,7 +283,11 @@ inline std::string fastIpv4ToString(
 }
 
 inline std::string fastIpv6ToString(const in6_addr& in6Addr) {
+#ifdef _MSC_VER
+  const uint16_t* bytes = reinterpret_cast<const uint16_6*>(&in6Addr.u.Word);
```

Comment:
Yes, it is. Looks like it happened when I was extracting the changes into separate PRs.